### DISCUSSION TITLE
Порядок расположения полей на панели

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#336] [Как это сделать?] [answered]
Есть возможность управлять порядком расположения полей на панели?
![image](https://user-images.githubusercontent.com/114401808/226594739-534563c4-26fa-4a38-ae40-d0298b2ef6b6.png)
![image](https://user-images.githubusercontent.com/114401808/226594878-7fdeb459-289c-4abe-abb2-c2708a690c65.png)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Порядок здесь в порядке возрастания `sort` полей.

Можно не показывать какие-то поля — параметр `hide` функции `setFormat` форматирования поля.

При открытии через `linkToPanel` можно передать видимость полей глобально — параметр `fields` (нет в доках еще, принимает `list` из `name` полей).

Еще в `linkToPanel` теперь можно принудительно показать только одну колонку через `columns: 1`.

И там же в `linkToPanel` можно передать кастомные названия полей, параметр `titles`: 

```
titles: json`[{"name":"select_1", "title":"Проверка"}]`;
```

**Reply by vldekov** [reply]
Через linkPanel параметр fields есть возможность устанавливать сортировку полей в одну колонку?

**Reply by ttm-alexeypolunin** [reply]
параметр `columns: 1`

**Reply by tiaga** [reply]
@ttm-alexeypolunin подскажите, с какой версии стал доступен параметр `columns` для `linkToPanel`?

**Reply by ttm-alexeypolunin** [reply]
Вот с той, которая сейчас последняя в мастере — **4.9.52.6**

**Comment by vldekov** [comment]
Да-- Отправлено из мобильного приложения Яндекс Почты21:24, 27 марта 2023 г., Alexey Polunin ***@***.***>:
Вот с той, которая сейчас последняя в мастере — 4.9.52.6
